aikau/src/main/resources/webscript-libs/service-filtering.lib.js

```diff
@@ -73,5 +77,19 @@ function alfAddUniqueServices(currentServices, servicesToFilter) {
          }
       }
    }
+   else if (currentServices && currentServices.length === 0 &&
+            servicesToFilter && servicesToFilter.length)
+   {
+      // If there are no currentServices, but there are servicesToFilter, then just return the complete
+      // list of servicesToFilter
+      filteredServices = servicesToFilter;
+   }
+   else if (currentServices && currentServices.length &&
+            servicesToFilter && servicesToFilter.length === 0 &&
+            includeCurrentServices !== false)
+   {
+      // If there ARE currentServices but no servicesToFilter, and we want to include the currentServices
+      // then just return them
```

Comment:
Don't understand this comment. "Return them" suggests that something is being returned, but the `filteredServices` variable is still set to its initial empty array, and so "nothing" will be returned.